app/templates/components/wds-global-navigation/link-button.hbs

```diff
@@ -0,0 +1,3 @@
+{{#wds-button class='wds-global-navigation__start-a-wiki-button' href=model.href secondary=true}}
```

Comment:
can we rename this class to `__link-button`? in theory this does not have to be start a wiki